apiserver/allfacades.go

```diff
@@ -252,6 +250,13 @@ func AllFacades() *facade.Registry {
 	reg("UserManager", 1, usermanager.NewUserManagerAPI)
 	reg("UserManager", 2, usermanager.NewUserManagerAPI) // Adds ResetPassword
 
+	// CAAS related facades.
+	// Move these to the correct place above once the feature flag disappears.
+	if featureflag.Enabled(feature.CAAS) {
+		reg("Cloud", 2, cloud.NewFacadeV2)
```

Comment:
I think it would be best to keep the registration of cloud v2 next to cloud v1, to avoid surprises down the line.

Reply:
ok